stdlib/universe/columns_test.flux

```diff
@@ -30,20 +28,21 @@ outData = "
 #group,false,false,true,true,true,true,false
 #default,_result,,,,,,
 ,result,table,_start,_stop,host,name,_value
+,,0,2018-05-20T19:53:26Z,2030-01-01T00:00:00Z,host.local,disk0,_field
```

Comment:
Curious why the row order changed here. Is the test not deterministic? Maybe it could use a call to `sort()` to make it so.

Reply:
That's a good note. Thanks for pointing it out. I added a sort so that this test is more consistent.